ui/raidboss/data/05-shb/trial/emerald_weapon-ex.js

```diff
@@ -220,40 +220,74 @@ export default {
       response: Responses.goRight(),
     },
     {
-      // TODO: use headmarkers for this
+      id: 'EmeraldEx Emerald Crusher',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '5585', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Smaragd-Waffe', id: '5585', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Émeraude', id: '5585', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'エメラルドウェポン', id: '5585', capture: false }),
+      response: Responses.knockback(),
+    },
+    {
+      id: 'EmeraldEx Secundus Terminus est Corner',
+      netRegex: NetRegexes.ability({ source: 'Bitblade', id: '55C9', capture: false }),
+      alarmText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Go to intercards',
+          de: 'in die Intercardinale Himmelsrichtungen',
+          fr: 'en intercadinal',
+          cn: '去场边四角',
+          ko: '대각위치로',
+        },
+      },
+    },
+    {
       id: 'EmeraldEx Secundus Terminus est',
-      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55C8', capture: false }),
-      netRegexDe: NetRegexes.startsUsing({ source: 'Smaragd-Waffe', id: '55C8', capture: false }),
-      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Émeraude', id: '55C8', capture: false }),
-      netRegexJa: NetRegexes.startsUsing({ source: 'エメラルドウェポン', id: '55C8', capture: false }),
+      netRegex: NetRegexes.ability({ source: 'Bitblade', id: '55CA', capture: false }),
```

Comment:
Were you able to run with this since updating it to `ability`? I'm curious if the callout comes out soon enough to react.

Reply:
just run it three times and it wasn't good enough.. I'm reverting it now

Reply:
That's what I was afraid of 😞. Maybe there's headMarkers for them? But that seems like a lot + also something we can do in a follow-up.

Reply:
yea, lets do it in a follow up so far I didn't saw anything.